app/dto/person.js

```diff
@@ -1,3 +1,20 @@
+const addContacts = (contacts) => {
+  const emails = contacts.filter(entry => entry.contact.contact_type.contact_type === 'Email')
+    .sort((a, b) => b.id - a.id)
+
+  const primaryTelephones = contacts.filter(entry => entry.contact.contact_type.contact_type === 'Phone')
+    .sort((a, b) => b.id - a.id)
+
+  const secondaryTelephones = contacts.filter(entry => entry.contact.contact_type.contact_type === 'SecondaryPhone')
```

Comment:
I assumed we could unlimited 'Phone' numbers, just displayed in 'id' order. This limits us to 2 phones - are you happy with that? I think the UI can show multiple (more than 2) phones if they exist

Reply:
We can store unlimited phone numbers. It's just a bit tricky as the user can only ever insert two phone numbers at the moment on the update page.

We also need the types to separated for that screen.